draft-ietf-ppm-dap.md

```diff
@@ -1279,10 +1270,45 @@ struct {
   };
 } PartialBatchSelector;
 
+enum {
+  initialize(0),
+  continue(1),
+  finish(2),
+  finished(3),
+  reject(4),
+  (255)
+} PrepareStepState;
+
+struct {
+  PrepareStepState prepare_step_state;
+  select (PrepareStep.prepare_step_state) {
+    case initialize: {
+      ReportShare report_share;
+      opaque leader_prep_share<0..2^32-1>;
```

Comment:
Hmm, I kind of like calling out that this will always be the leader's preparation share; thoughts on keeping this as-is / maybe there's a great reason to rename that I'm overlooking?

Reply:
Actually, feel free to disrgard. I think we've moved beyond where this suggestion is helpful :)